### ISSUE TITLE
Allow `placeItems center` without forcing usage of `placeItems2 center center`

### ISSUE THREAD

**Issue by jigargosar** [dzuk-mutant/elm-css#9]
IMHO, this can be achieved by expanding type signature of `placeItems` to allow for `alignItems` values
https://github.com/dzuk-mutant/elm-css/blob/96625ede2e57db18f13c2faa37f965685b7e1fb3/src/Css.elm#L11145-L11149

This is also specified in the spec:
https://developer.mozilla.org/en-US/docs/Web/CSS/place-items#formal_syntax
```
place-items = 
  <'align-items'> <'justify-items'>[?](https://developer.mozilla.org/en-US/docs/Web/CSS/Value_definition_syntax#question_mark)
```

It's already specified in the signature of `placeItems2` (1st argument).

https://github.com/dzuk-mutant/elm-css/blob/96625ede2e57db18f13c2faa37f965685b7e1fb3/src/Css.elm#L11164-L11202

**Comment by jigargosar** [comment]
FYI: following is the workaround I am currently using. And many thanks for this amazing work. Your work on elm-css made me switch back to it.

```elm
placeItems :
    BaseValue
        { normal : Supported
        , stretch : Supported
        , center : Supported
        , start : Supported
        , end : Supported
        , flexStart : Supported
        , flexEnd : Supported
        , selfStart : Supported
        , selfEnd : Supported
        , baseline : Supported
        , firstBaseline : Supported
        , lastBaseline : Supported
        }
    -> Style
placeItems value =
    property "place-items" (Css.Value.unpack value)


```

**Comment by dzuk-mutant** [comment]
Thanks! And yeah, I think this makes sense (haven't used `place-items` in a while). I'll check on this and add it in when I can!

**Comment by dzuk-mutant** [comment]
Added it to the latest commit! It'll be in whatever the next release is. (Should be really soon)

**Comment by dzuk-mutant** [comment]
Okay, published in the latest 3.0.0 release ^^
